modules/ROOT/pages/dw-core-functions-orderby.adoc

```diff
@@ -151,9 +152,77 @@ output application/json
 ]
 ----
 
+=== Example
+
+This example orders a JSON array payload descending by the type String. The expression first orders them alphabetically by the value of the `Type` key, then reverses the order based on the `[-1 to 0]`.
```

Comment:
"This example orders a JSON array payload descending by the type String." -> "This example orders the objects in a JSON array. 

* removes "payload descending by the type String" 
The value of a DW variable is not the same as the payload. Notice that the body of the script uses myInput (not payload).